_test/test_sqw/test_PixelData.m

```diff
@@ -1091,20 +1093,6 @@ function test_has_more_is_true_after_appending_page_to_non_file_backed(obj)
         assertFalse(pix.has_more());
     end
 
-    function test_max_page_size_set_by_pixel_page_size_config_option(obj)
```

Comment:
this test needs to be replaced with a pair of "page size is set by constructor argument" and "page size is infinite be default".

A test demonstrating the new patterns of
```
page_size = get(hor_config, 'pixel_page_size');
s = sqw('filename', 'pixel-page-size', page_size)
```
would be a useful addition to help developers and test the config file contains a sensible, non-infinite value there.

Reply:
First test already exists, I've added the second.